app/frontend/src/components/apiAccess/Step4.vue

```diff
@@ -1,21 +1,70 @@
 <template>
   <v-container>
-    <h2>API Access for Common Services</h2>Contents to go here
-    <br />
-    <br />
-    <br />
-    <br />
-    <div class="text-center">
-      <v-btn class="BC-Gov-PrimaryButton" text @click="finish">Finish</v-btn>
+    <h2 class="green--text text--darken-2 text-center mb-12">
+      <v-icon large color="green darken-2">check_circle</v-icon>API Documentation and Usage
+    </h2>
+    <p class="mb-12">
+      This service client can be used to make REST API calls and an example Postman collection is provided.
+      <br />Fetch a new token using the service client and password against the token endpoint (see postman collection) and use that token in your bearer header.
+    </p>
+
+    <div>
+      <ul>
+        <li v-for="item in KeycloakCommonServiceList" v-bind:key="item.name" class="mb-5">
+          {{item.name}}
+          <br />
+          <v-row>
+            <v-col cols="6">
+              <a :href="item.apiDocLink" target="_blank">
+                <strong>API DOCS</strong>
+                <v-btn text large icon class="BC-Gov-IconButton">
+                  <v-icon>open_in_new</v-icon>
+                </v-btn>
+              </a>
+            </v-col>
+            <v-col cols="6">
+              <a
+                class="buttonLink"
+                :href="`/files/${item.postmanCollection}`"
+                :download="item.postmanCollection"
+                target="_blank"
+              >
+                <strong>Postman Collection</strong>
+                <v-btn text large icon class="BC-Gov-IconButton">
+                  <v-icon>cloud_download</v-icon>
+                </v-btn>
+              </a>
+            </v-col>
+          </v-row>
+        </li>
+      </ul>
     </div>
+
+    <v-row class="mt-12">
+      <v-col cols="6" offset="3">
+        <v-btn class="BC-Gov-PrimaryButton" block text @click="finish">Finish</v-btn>
+      </v-col>
+    </v-row>
   </v-container>
 </template>
 
 <script>
 import { mapMutations } from 'vuex';
 
+import {
+  CommonServiceList,
+  CommonServiceTypes
+} from '@/utils/commonServices.js';
+
 export default {
   name: 'ApiAccessStep4',
+  data() {
+    return {
+      KeycloakCommonServiceList: CommonServiceList.filter(
```

Comment:
I'm half tempted to say this should be a computed instead of data as we don't directly manipulate this piece of data at all.

Reply:
Doesn't work in a computed. Getting the import in as something referrable in computed and methods/etc requires putting it in the data block anywyas, so it's one-lined there.